### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by hemanthkaipaa** [android/ndk#1095]
--------------------------------------------------------------------------------

#### Description
App Crash while loading with below message. in 5.1 version OPPO Device.
	10-09 15:09:59.547 14701-14762/com.arteriatech.ss.bil D/shim: loading shared libraries
	10-09 15:09:59.547 14701-14762/com.arteriatech.ss.bil W/linker: /data/app/com.arteriatech.ss.bil-1/lib/arm64/libodataofflinejni.so: unused DT entry: type 0x6ffffffe arg 0x78b98
	10-09 15:09:59.547 14701-14762/com.arteriatech.ss.bil W/linker: /data/app/com.arteriatech.ss.bil-1/lib/arm64/libodataofflinejni.so: unused DT entry: type 0x6fffffff arg 0x3

* NDK Version:
* Build system: 
* Host OS: Windows
* ABI:arm64-v8a
* NDK API level:
* Device API level:5.1

**Comment by hemanthkaipaa** [comment]
[errorlog - Copy.txt](https://github.com/android/ndk/files/3707115/errorlog.-.Copy.txt)

**Comment by enh-google** [comment]
no, those are just warnings (that's what the "W/" before "linker" means --- errors would be "E/").

seems like you've cut off the error log just before it was about to get interesting? where's the actual tombstone? add that here if you have it, otherwise closing this because there's not enough info to go on.
